AnyText/Tests/AnyTextGrammarServer/Program.cs

```diff
@@ -9,9 +9,11 @@
 }
 
 var grammar = new AnyTextGrammar();
-var lspServer = new LspServer(grammar);
-using (var rpc = AnyTextJsonRpcServerUtil.CreateServer(FullDuplexStream.Splice(Console.OpenStandardInput(), Console.OpenStandardOutput()), lspServer))
+using (var rpc = AnyTextJsonRpcServerUtil.CreateServer(FullDuplexStream.Splice(Console.OpenStandardInput(), Console.OpenStandardOutput())))
 {
+    var lspServer = new LspServer(rpc, grammar);
+    rpc.AddLocalRpcTarget(lspServer, AnyTextJsonRpcServerUtil.CreateTargetOptions());
```

Comment:
Achtung, jetzt geht das Binding über die Klasse, nicht über die Schnittstelle. Ich fände es über die Schnittstelle aber schöner, dann müssen wir es jetzt aber explizit machen, also `AddLocalRpcTarget<ILspServer>`

Reply:
Ist jetzt eine Methode in AnyTextJsonRpcServerUtil. Damit ist das Binding üver die Schnittstelle und CreateTargetOptions ist wieder private.